bgpd/bgp_vty.c

```diff
@@ -14977,20 +14978,21 @@ ALIAS (show_lcommunity_list,
 
 DEFUN (show_lcommunity_list_arg,
        show_bgp_lcommunity_list_arg_cmd,
-       "show bgp large-community-list <(1-500)|WORD>",
+       "show bgp large-community-list <(1-500)|WORD> detail",
        SHOW_STR
        BGP_STR
        "List large-community list\n"
-       "large-community-list number\n"
-       "large-community-list name\n")
+       "Large-community-list number\n"
```

Comment:
This should be lowercased `large-community-list` because it's a command, not a description.

Reply:
But I see others are used the same way :/ Then maybe Ok.